.gitignore

```diff
@@ -21,4 +21,6 @@
 
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
\ No newline at end of file
+yarn-error.log*
+
+.vscode/settings.json
```

Comment:
I appreciate that you added this. I think this will help everyone be able to customize their VSCode in their own way without being impacted by others.